source/extensions/retry/host/omit_hosts/config.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "envoy/config/retry/omit_hosts/v2/omit_hosts_config.pb.validate.h"
+#include "envoy/upstream/retry.h"
+
+#include "extensions/retry/host/omit_hosts/omit_hosts.h"
+#include "extensions/retry/host/well_known_names.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Retry {
+namespace Host {
+
+class OmitHostsRetryPredicateFactory : public Upstream::RetryHostPredicateFactory {
+public:
+  Upstream::RetryHostPredicateSharedPtr createHostPredicate(const Protobuf::Message&,
```

Comment:
we typically include the parameter names in  the function prototype

Reply:
Done